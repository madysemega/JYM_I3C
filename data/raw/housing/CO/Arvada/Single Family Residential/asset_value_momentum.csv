predictions_asset_value_momentum
-0.04803221526492854
-0.07202047169503974
-0.02492301635778614
-0.15591357906793488
0.029887569191456773
-0.060127571421844356
-0.13432181787907083
0.023544629566381366
-0.07451757034474941
0.08320070209114747
0.01471718902241829
-0.16580258363748374
-0.05967562336765157
-0.15248210708003224
-0.12952857758321742
-0.11399264909460403
-0.09476726279340704
-0.17712266187831585
-0.12104063339423983
-0.14853644782390393
-0.19446720339902307
0.0050530164760385935
-0.4607512256398816
-0.1456116330158319
-0.14606051193447034
-0.16132542861610805
-0.1398152850204167
