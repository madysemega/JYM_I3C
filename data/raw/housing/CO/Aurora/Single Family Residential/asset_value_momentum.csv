predictions_asset_value_momentum
-10.868872298279358
0.3623801188063955
-9.740059361411731
-6.849217604821828
-2.8633236323339424
25.374875154698785
-2.6549780787303074
-3.5715913101371215
-9.372960077537902
-11.552203005930053
-0.6169846075903433
-4.860282104899952
-10.049637164921648
-0.037038919351738996
-12.806455998991572
-5.725014025242852
-6.949518868722067
-9.110643377150499
-8.857141747058057
-2.8956368762265963
-6.206654969687412
-7.900350718644622
-9.750610536146862
-8.750199883674169
-13.78294747253983
-9.817832500070223
-16.75049683368807
