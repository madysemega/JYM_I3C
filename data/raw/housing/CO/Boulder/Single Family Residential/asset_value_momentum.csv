predictions_asset_value_momentum
0.19298908356608435
-0.24710188176568096
0.0013858624172350524
0.28145121270995305
0.07863009076262481
0.055303521629749615
0.2765896518216189
-0.4275794601822713
0.3840459947698286
0.16384877508340454
-0.18130630886268997
-0.27293632987467814
0.08656570123644847
0.7208161947720608
0.2497726546633882
-0.10671351783574919
0.5716984120286054
-0.7751334728068255
0.7233320849888905
-0.36395940280450084
-0.7421072489078435
0.18097696227404259
0.46115173941732673
0.4484669830212401
0.35073736819599804
0.811034717602135
-0.0984897087733818
