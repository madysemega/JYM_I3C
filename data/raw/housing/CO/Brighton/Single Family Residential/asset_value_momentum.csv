predictions_asset_value_momentum
0.057109385941987965
0.8940032133527456
-0.44009655889168825
0.03439219767406797
0.13567277321713325
0.6698655725475813
0.2979766094418606
-0.010827377273345944
0.3845359564769974
0.06773492560168948
0.48740585593106134
1.3672609950604706
0.2014721386326152
0.5379749573537161
0.12480496049955611
0.0874008378771816
0.36647970061456364
-0.18257371782126658
0.254418154729385
0.43889630141703884
0.3095663163492324
0.5214978591512511
0.3005194373327827
1.5098341261031423
0.14904582128849095
0.7913730566896343
-0.035074925196291525
