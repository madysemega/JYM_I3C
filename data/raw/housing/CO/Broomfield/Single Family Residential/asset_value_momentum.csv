predictions_asset_value_momentum
0.012546952432447892
-0.0811925276501743
-0.06402879039621528
0.0038259690154121492
-0.017728302916750642
-0.06610360839715354
0.025360290774488998
-0.011046461263954633
-0.04184942088803001
-0.051980678408315155
-0.01095860247742442
-0.05014144195227309
-0.048502850696215426
0.011079491420372902
-0.024112122884139046
0.03193027424170872
-0.01331624747698636
-0.06288018238648894
0.04141155700621469
0.0896902763401484
-0.03070917966446427
-0.044530640744797775
-0.05568446907208354
0.0524468143115318
-0.014712430004842956
-0.07068480646250977
-0.1108906169585528
