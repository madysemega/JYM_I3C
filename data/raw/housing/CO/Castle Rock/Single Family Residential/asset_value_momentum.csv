predictions_asset_value_momentum
0.12572012018750992
0.056024259175587524
0.15261473137529027
0.0006162252847529989
0.003920367135866757
0.09931162075736182
0.30822804232622864
-0.10477535964923997
-0.05320986771123055
0.19019535858444306
-0.2303769254799853
-0.17669098536337746
0.12765912566851317
-0.09676198541174784
-0.012530558369420498
-0.02107151556774858
0.013368710962383002
0.06570088424948033
-0.005147134182024173
0.03301316645751546
-0.06069889093851415
-0.1384419050188483
-0.08300881526587936
0.22727298874288981
-0.054452091020235543
0.0852273075885364
-0.2203948934389607
