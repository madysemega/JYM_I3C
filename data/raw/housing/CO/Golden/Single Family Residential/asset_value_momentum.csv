predictions_asset_value_momentum
0.4168906907916211
-0.07350361560845584
-0.3197650682271413
-0.0734198666228699
0.026083092275974922
-0.05529622976340873
0.6631003200131975
-0.4712936399578482
0.41725521525167913
0.41446188535756623
-0.05168293196432139
0.8954167816502455
-0.5606720693328585
0.2567810856748604
0.8179650070230369
0.30759843430608413
-0.10196384585707056
0.37927712916398054
0.09074206086382082
-0.10416117486490613
0.045318418430931824
0.692579284122952
0.6230336004095941
0.879268925063701
0.017592540356332887
-0.3384116483471558
0.10252393943178437
